### PR TITLE
[EDU-2450] BUILD MENU PT functions instances

### PR DESCRIPTION
**Pull request by bru-andrade**
### Changes

- Translated functions instances guide

### Additional links

[EDU-2450]: https://aziontech.atlassian.net/browse/EDU-2450?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-functions-instances/functions-instances.mdx [hunk 1]
```diff
@@ -0,0 +1,204 @@
+---
+title: Como instanciar edge functions em sua aplicação
+description: Instantiate serverless functions directly within your edge application.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-functions-instances/functions-instances.mdx:3]:
```suggestion
description: Instancie funções serverless diretamente na sua edge application.
```

File: src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-functions-instances/functions-instances.mdx [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+meta_tags: build
+namespace: docs_guides_build_instantiate_edge_functions
+permalink: /documentation/products/guides/build/instantiate-edge-functions/
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-functions-instances/functions-instances.mdx:6]:
```suggestion
permalink: /documentacao/produtos/guias/build/instanciar-edge-functions/
```

File: src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-functions-instances/functions-instances.mdx [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+menu_namespace: buildMenu
+---
+
+Instancie [funções serverless](/en/documentation/products/edge-application/edge-functions-instances/) diretamente dentro de sua edge application.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-functions-instances/functions-instances.mdx:10]:
```suggestion
Instancie [funções serverless](/pt-br/documentacao/produtos/edge-application/edge-functions-instances/) diretamente dentro de sua edge application.
```

File: src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-functions-instances/functions-instances.mdx [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+
+:::caution[warning]
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-functions-instances/functions-instances.mdx:12]:
```suggestion
:::caution[Atenção]
```

File: src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-functions-instances/functions-instances.mdx [hunk 5]
```diff
@@ -0,0 +1,204 @@
...
+Você deve ativar o módulo [Edge Functions](/en/documentation/products/edge-application/edge-functions/) para implementar funções em sua aplicação. Se **Edge Functions** estiver ativado, o tempo de computação e as invocações podem gerar custos relacionados ao uso. Consulte a [página de preços](https://www.azion.com/en/pricing/) para mais informações.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-functions-instances/functions-instances.mdx:13]:
```suggestion
Você deve ativar o módulo [Edge Functions](/pt-br/documentacao/produtos/edge-application/edge-functions/) para implementar funções em sua aplicação. Se **Edge Functions** estiver ativado, o tempo de computação e as invocações podem gerar custos relacionados ao uso. Consulte a [página de preços](https://www.azion.com/pt-br/precos/) para mais informações.
```

File: src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-functions-instances/functions-instances.mdx [hunk 6]
```diff
@@ -0,0 +1,204 @@
...
+:::
+
+Este guia usa a função pré-configurada *Azion - Hello World*, uma função global incluída em todas as contas Azion. Você pode repetir este processo usando qualquer função já disponível no [Azion Marketplace](https://marketplace.azion.com/) ou [desenvolver sua própria função](/en/documentation/products/guides/edge-functions/api-builder/)
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-functions-instances/functions-instances.mdx:16]:
```suggestion
Este guia usa a função pré-configurada *Azion - Hello World*, uma função global incluída em todas as contas Azion. Você pode repetir este processo usando qualquer função já disponível no [Azion Marketplace](https://marketplace.azion.com/) ou [desenvolver sua própria função](/pt-br/documentacao/produtos/guias/edge-functions/api-builder/)
```

File: src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-functions-instances/functions-instances.mdx [hunk 7]
```diff
@@ -0,0 +1,204 @@
...
+5. Clique no botão **Save**.
+6. Vá para a aba **Functions**.
+7. Clique em **Add Function**.
+8. Nomeie sua instância de função. Por exemplo: `Hello World function`.
+9. Selecione a função **Azion - Hello World**.
+10. Clique no botão **Save**.
+
+A página de funções agora lista a nova instância criada. No entanto, esta nova função ainda não está ativa em sua aplicação. Você precisa definir o que acionará a função em sua aplicação. Para invocar a função instanciada no URI `xxxxxxxxxx.map.azionedge.net/hello-world`:
+
+11. Ainda na página **Edge Application**, navegue até a aba **Rules Engine**.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-functions-instances/functions-instances.mdx:35]:
```suggestion
11. Ainda na página do **Edge Application**, navegue até a aba **Rules Engine**.
```

File: src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-functions-instances/functions-instances.mdx [hunk 8]
```diff
@@ -0,0 +1,204 @@
...
+12. Clique no botão **Add Rule** e selecione **Request Phase**.
+13. Na seção **Criteria**, selecione a variável `${uri}`.
+
+:::note
+A variável `${uri}` pode já estar selecionada por padrão se você não ativou o módulo [Application Acceleration](/en/documentation/products/edge-application/application-acceleration/).
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-functions-instances/functions-instances.mdx:40]:
```suggestion
A variável `${uri}` pode já estar selecionada por padrão se você não ativou o módulo [Application Acceleration](/pt-br/documentacao/produtos/edge-application/application-acceleration/).
```

File: src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-functions-instances/functions-instances.mdx [hunk 9]
```diff
@@ -0,0 +1,204 @@
...
+:::
+
+14. Como operador de comparação, selecione **is equal**.
+15. Como argumento, adicione `/hello-world`.
+16. Na seção **Behaviors**, selecione **Run Function** da lista de comportamentos.
+17. Selecione a função **Hello World** instanciada.
+18. Clique no botão **Save**.
+
+Para ver sua função em execução, acesse sua aplicação [usando seu domínio](/en/documentation/products/guides/configure-a-domain/) e o URI definido na regra, que está no formato `xxxxxxxxx.map.azionedge.net/hello-world`.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-functions-instances/functions-instances.mdx:49]: link pt

File: src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-functions-instances/functions-instances.mdx [hunk 10]
```diff
@@ -0,0 +1,204 @@
...
+
+## Via API
+
+1. Execute a seguinte solicitação `PATCH` em seu terminal, substituindo `[TOKEN VALUE]` por seu [personal token](/en/documentation/products/guides/personal-tokens/) e a variável `<edge_application_id>` por [seu ID de edge application](/en/documentation/products/guides/build/configure-main-settings/) para ativar o módulo [Edge Functions](/en/documentation/products/edge-application/edge-functions/):
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-functions-instances/functions-instances.mdx:53]:
```suggestion
1. Execute a seguinte requisição `PATCH` em seu terminal, substituindo `[TOKEN VALUE]` por seu [personal token](/pt-br/documentacao/produtos/gestao-de-contas/personal-tokens/) e a variável `<edge_application_id>` por [seu ID de edge application](/pt-br/documentation/products/guides/build/definir-configuracoes-principais/) para ativar o módulo [Edge Functions](/pt-br/documentacao/produtos/edge-application/edge-functions/):
```

File: src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-functions-instances/functions-instances.mdx [hunk 11]
```diff
@@ -0,0 +1,204 @@
...
+      "reference_count": 4,
+      "version": "-",
+      "vendor": "azionmanager@azion.com"
+    }
+  ]
+}
+```
+
+5. Copie o valor `<edge_function_id>`.
+6. Execute a seguinte solicitação `POST`, substituindo o valor do ID da edge function pelo valor que você recebeu na resposta anterior:
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-functions-instances/functions-instances.mdx:101]:
```suggestion
6. Execute a seguinte requisição `POST`, substituindo o valor do ID da edge function pelo valor que você recebeu na resposta anterior:
```

File: src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-functions-instances/functions-instances.mdx [hunk 12]
```diff
@@ -0,0 +1,204 @@
...
+```
+
+  | Chave | Descrição |
+  | --- | --- |
+  | `edge_function_id` | O ID da edge function. Este não é o mesmo que o ID da função instanciada, que é único para cada instância de função da aplicação. |
+  | `name` | Nome da função instanciada. |
+  | `args` | Definição dos argumentos necessários para a função. Algumas funções não necessitam de argumentos. |
+  | `id` | ID da função instanciada. |
+  
+8. Execute a seguinte solicitação `POST` em seu terminal, substituindo `[TOKEN VALUE]` por seu [personal token](/en/documentation/products/guides/personal-tokens/), a variável `<edge_application_id>` por [seu ID de edge application](/en/documentation/products/guides/build/configure-main-settings/), e a variável `<edge_function_instance_id>` pelo ID da instância da função recebida na resposta anterior:
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-functions-instances/functions-instances.mdx:135]:
```suggestion
8. Execute a seguinte requisição `POST` em seu terminal, substituindo `[TOKEN VALUE]` por seu [personal token](/pt-br/documentacao/produtos/gestao-de-contas/personal-tokens/), a variável `<edge_application_id>` por [seu ID de edge application](/pt-br/documentation/products/guides/build/definir-configuracoes-principais/), e a variável `<edge_function_instance_id>` pelo ID da instância da função recebida na resposta anterior:
```

File: src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-functions-instances/functions-instances.mdx [hunk 13]
```diff
@@ -0,0 +1,204 @@
...
+                "conditional": "if",
+                "input_value": "/hello-world"
+            }
+        ]
+    ]
+}'
+```
+
+:::note
+Para mais informações sobre o endpoint de regras, veja o guia [sobre como trabalhar com regras](/en/documentation/products/guides/build/work-with-rules-engine/).
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-functions-instances/functions-instances.mdx:164]: link pt

File: src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-functions-instances/functions-instances.mdx [hunk 14]
```diff
@@ -0,0 +1,204 @@
...
+        "is_active": true,
+        "order": 2,
+        "description": ""
+    }
+}
+```
+
+10. Aguarde alguns minutos para que as alterações se propaguem.
+
+Para ver sua função em execução, acesse sua aplicação [usando seu domínio](/en/documentation/products/guides/configure-a-domain/) e o URI definido na regra, que está no formato `xxxxxxxxx.map.azionedge.net/hello-world`.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-functions-instances/functions-instances.mdx:200]: link

File: src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-functions-instances/functions-instances.mdx (hunk 9 above)
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-functions-instances/functions-instances.mdx:49]:
```suggestion
Para ver sua função em execução, acesse sua aplicação [usando seu domínio](/pt-br/documentacao/produtos/guias/build/configurar-dominio/) e o URI definido na regra, que está no formato `xxxxxxxxx.map.azionedge.net/hello-world`.
```

File: src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-functions-instances/functions-instances.mdx [hunk 15]
```diff
@@ -0,0 +1,204 @@
...
+15. Como argumento, adicione `/hello-world`.
+16. Na seção **Behaviors**, selecione **Run Function** da lista de comportamentos.
+17. Selecione a função **Hello World** instanciada.
+18. Clique no botão **Save**.
+
+Para ver sua função em execução, acesse sua aplicação [usando seu domínio](/pt-br/documentacao/produtos/guias/configurar-dominio/) e o URI definido na regra, que está no formato `xxxxxxxxx.map.azionedge.net/hello-world`.
+
+## Via API
+
+1. Execute a seguinte requisição `PATCH` em seu terminal, substituindo `[TOKEN VALUE]` por seu [personal token](/pt-br/documentacao/produtos/gestao-de-contas/personal-tokens/) e a variável `<edge_application_id>` por [seu ID de edge application](/pt-br/documentation/products/guides/build/definir-configuracoes-principais/) para ativar o módulo [Edge Functions](/pt-br/documentacao/produtos/edge-application/edge-functions/):
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-functions-instances/functions-instances.mdx:53]:
```suggestion
1. Execute a seguinte requisição `PATCH` em seu terminal, substituindo `[TOKEN VALUE]` por seu [personal token](/pt-br/documentacao/produtos/guias/personal-tokens/) e a variável `<edge_application_id>` por [seu ID de edge application](/pt-br/documentation/products/guides/build/definir-configuracoes-principais/) para ativar o módulo [Edge Functions](/pt-br/documentacao/produtos/edge-application/edge-functions/):
```

**Pull request closed without merging**